pages/pricing.md

```diff
@@ -0,0 +1,8 @@
+---
+title: pricing
+description: a page for those that want to buy your products so they can know the cost
+creator_link: https://paul.af/pricing
```

Comment:
question: not sure if this should be this URL or https://paul.af/slash-pricing

Reply:
Yeah for creator links I try to link to the explainer page if I can